### PR TITLE
Added bigwig files

### PR DESCRIPTION
**Pull request by keyvanelhami**
Check list if changes affect bed files:

- [ ] Update shasum
- [ ] Update annotations
- [ ] Update filename if relevant
- [ ] Update person who designed/changed it
- [ ] Update bed file version
- [ ] Update UCSC track
- [ ] Update statusDB
- [ ] Deploy on Hasta
- [ ] Update LIMS

### CODE REVIEW

**Comment by keyvanelhami** [comment]
@hassanfa I have now uploaded the `.bw` files and an description `.md` files for each file. Could you just run a sanity check to see if the `.bw` files are in correct format+

File: ucsc_track_hub/hg19/docs/BigWig_gicfdna_3.1.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+
+# Display Conventions and Configuration
+
+No specific display conventions and configuration.
+
+
+# Methods
+
+Sample: HD778
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gicfdna_3.1.md:17]:
```suggestion
Sample: Horizon Discovery HD778
```

File: ucsc_track_hub/hg19/docs/BigWig_gicfdna_3.1.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+Lims-id: ACC5456A2
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gicfdna_3.1.md:19]:
```suggestion
Internal LIMS ID: ACC5456A2
```

File: ucsc_track_hub/hg19/docs/BigWig_gicfdna_3.1.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+Case-id: usablemagpie_10M
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gicfdna_3.1.md:21]:
```suggestion
Internal case ID: usablemagpie_10M
```

File: ucsc_track_hub/hg19/docs/BigWig_gicfdna_3.1.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+Lab Method: Twist (both lib prep and hybridization)
+
+Amount reads used for the analysis: 10M r-p
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gicfdna_3.1.md:25]:
```suggestion
Amount of reads used for the analysis: 10M r-p
```

File: ucsc_track_hub/hg19/docs/BigWig_gmcksolid_4.1.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+# Display Conventions and Configuration
+
+No specific display conventions and configuration.
+
+
+# Methods
+
+Sample: NA24143
+
+Lims-id: ACC5812A1
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gmcksolid_4.1.md:19]:
```suggestion
Internal LIMS ID: ACC5812A1
```

File: ucsc_track_hub/hg19/docs/BigWig_gmcksolid_4.1.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+Case-id: mintpossum_30M
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gmcksolid_4.1.md:21]:
```suggestion
Internal case ID: mintpossum_30M
```

File: ucsc_track_hub/hg19/docs/BigWig_gmcksolid_4.1.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+Lab Method: Twist (both lib prep and hybridization)
+
+Amount reads used for the analysis: 30M r-p
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gmcksolid_4.1.md:25]:
```suggestion
Amount of reads used for the analysis: 30M r-p
```

File: ucsc_track_hub/hg19/docs/BigWig_gicfdna_3.1.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+
+2) sort the copied file: `cat <cp.bedgraph> | sort -k1,1 -k2,2n > <cp.sorted.bedgraph>`
+
+3) Run: `bedGraphToBigWig <cp.sorted.bedgraph> /home/proj/production/cancer/reference/GRCh37/genome/hg19.chrom.sizes <output.bw>`.
+
+
+
+
+# External links
+
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gicfdna_3.1.md:62]:
```suggestion

Read more about BALSAMIC: https://github.com/Clinical-Genomics/BALSAMIC
```

File: ucsc_track_hub/hg19/docs/BigWig_gmcksolid_4.1.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+
+2) sort the copied file: `cat <cp.bedgraph> | sort -k1,1 -k2,2n > <cp.sorted.bedgraph>`
+
+3) Run: `bedGraphToBigWig <cp.sorted.bedgraph> /home/proj/production/cancer/reference/GRCh37/genome/hg19.chrom.sizes <output.bw>`.
+
+
+
+
+# External links
+
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gmcksolid_4.1.md:62]:
```suggestion

Read more about BALSAMIC: https://github.com/Clinical-Genomics/BALSAMIC
```

File: ucsc_track_hub/hg19/docs/BigWig_gmslymphoid_7.1.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+
+
+# Display Conventions and Configuration
+
+No specific display conventions and configuration.
+
+
+# Methods
+
+Sample: HD829
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gmslymphoid_7.1.md:17]:
```suggestion
Sample: Horizon Discovery HD829
```

File: ucsc_track_hub/hg19/docs/BigWig_gmslymphoid_7.1.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+
+Lims-id: ACC6648A1
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gmslymphoid_7.1.md:19]:
```suggestion
Internal LIMS ID: ACC6648A1
```

File: ucsc_track_hub/hg19/docs/BigWig_gmslymphoid_7.1.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+
+Case-id: soundmarlin
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gmslymphoid_7.1.md:21]:
```suggestion
Internal case ID: soundmarlin
```

File: ucsc_track_hub/hg19/docs/BigWig_gmslymphoid_7.1.md [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+
+2) sort the copied file: `cat <cp.bedgraph> | sort -k1,1 -k2,2n > <cp.sorted.bedgraph>`
+
+3) Run: `bedGraphToBigWig <cp.sorted.bedgraph> /home/proj/production/cancer/reference/GRCh37/genome/hg19.chrom.sizes <output.bw>`.
+
+
+
+
+# External links
+
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gmslymphoid_7.1.md:62]:
```suggestion

Read more about BALSAMIC: https://github.com/Clinical-Genomics/BALSAMIC
```

File: ucsc_track_hub/hg19/docs/BigWig_gmslymphoid_7.1.md [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+Sample: HD829
+
+Lims-id: ACC6648A1
+
+Case-id: soundmarlin
+
+Lab Method: KAPA + Twist
+
+Amount reads used for the analysis: 17M r-p
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gmslymphoid_7.1.md:25]:
```suggestion
Amount of reads used for the analysis: 17M r-p
```

File: ucsc_track_hub/hg19/docs/BigWig_gmsmyeloid_5.2.md [hunk 15]
```diff
@@ -0,0 +1,69 @@
...
+# Display Conventions and Configuration
+
+No specific display conventions and configuration.
+
+
+# Methods
+
+Sample: NA24143
+
+Lims-id: ACC6306A1
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gmsmyeloid_5.2.md:19]:
```suggestion
Internal LIMS ID: ACC6306A1
```

File: ucsc_track_hub/hg19/docs/BigWig_gmsmyeloid_5.2.md [hunk 16]
```diff
@@ -0,0 +1,69 @@
...
+
+Case-id: amusedghoul
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gmsmyeloid_5.2.md:21]:
```suggestion
Internal case ID: amusedghoul
```

File: ucsc_track_hub/hg19/docs/BigWig_gmsmyeloid_5.2.md [hunk 17]
```diff
@@ -0,0 +1,69 @@
...
+
+Lab Method: KAPA + Twist
+
+Amount reads used for the analysis: 30M r-p
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gmsmyeloid_5.2.md:25]:
```suggestion
Amount of reads used for the analysis: 30M r-p
```

File: ucsc_track_hub/hg19/docs/BigWig_gmsmyeloid_5.2.md [hunk 18]
```diff
@@ -0,0 +1,69 @@
...
+
+Bioinformatic method: Analyzed with Balsamic 4.0.0 and .bw-file is created with mosdepth
+
+### How the file was generated:
+
+Below procedure can be used for several different panel-designs.
+
+Activate an conda environment with following tools: mosdepth and bedGraphToBigWig. E.g: `source activate D_BALSAMIC-base_develop`
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gmsmyeloid_5.2.md:33]:
```suggestion
Activate a Conda environment with following tools: mosdepth and bedGraphToBigWig. E.g: `source activate D_BALSAMIC-base_develop`
```

File: ucsc_track_hub/hg19/docs/BigWig_gmsmyeloid_5.2.md [hunk 19]
```diff
@@ -0,0 +1,69 @@
...
+
+2) sort the copied file: `cat <cp.bedgraph> | sort -k1,1 -k2,2n > <cp.sorted.bedgraph>`
+
+3) Run: `bedGraphToBigWig <cp.sorted.bedgraph> /home/proj/production/cancer/reference/GRCh37/genome/hg19.chrom.sizes <output.bw>`.
+
+
+
+
+# External links
+
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gmsmyeloid_5.2.md:62]:
```suggestion

Read more about BALSAMIC: https://github.com/Clinical-Genomics/BALSAMIC
```

File: ucsc_track_hub/hg19/docs/BigWig_gmslymphoid_7.1.md [hunk 20]
```diff
@@ -0,0 +1,69 @@
...
+
+Bioinformatic method: Analyzed with Balsamic 4.2.4 and .bw-file is created with mosdepth
+
+### How the file was generated:
+
+Below procedure can be used for several different panel-designs.
+
+Activate an conda environment with following tools: mosdepth and bedGraphToBigWig. E.g: `source activate D_BALSAMIC-base_develop`
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gmslymphoid_7.1.md:33]:
```suggestion
Activate a Conda environment with following tools: mosdepth and bedGraphToBigWig. E.g: `source activate D_BALSAMIC-base_develop`
```

File: ucsc_track_hub/hg19/docs/BigWig_gmcksolid_4.1.md [hunk 21]
```diff
@@ -0,0 +1,69 @@
...
+Amount reads used for the analysis: 30M r-p
+
+Bioinformatic method: Analyzed with Balsamic 3.0.0 and .bw-file is created with mosdepth
+
+### How the file was generated:
+
+Below procedure can be used for several different panel-designs.
+
+Activate an conda environment with following tools: mosdepth and bedGraphToBigWig. E.g: `source activate D_BALSAMIC-base_develop`
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gmcksolid_4.1.md:33]:
```suggestion
Activate a Conda environment with following tools: mosdepth and bedGraphToBigWig. E.g: `source activate D_BALSAMIC-base_develop`
```

File: ucsc_track_hub/hg19/docs/BigWig_gicfdna_3.1.md [hunk 22]
```diff
@@ -0,0 +1,69 @@
...
+Amount reads used for the analysis: 10M r-p
+
+Bioinformatic method: Analyzed with Balsamic 4.2.4 and .bw-file is created with mosdepth
+
+### How the file was generated:
+
+Below procedure can be used for several different panel-designs.
+
+Activate an conda environment with following tools: mosdepth and bedGraphToBigWig. E.g: `source activate D_BALSAMIC-base_develop`
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gicfdna_3.1.md:33]:
```suggestion
Activate a Conda environment with following tools: mosdepth and bedGraphToBigWig. E.g: `source activate D_BALSAMIC-base_develop`
```

File: ucsc_track_hub/hg19/docs/BigWig_lymphoma_6.1.md [hunk 23]
```diff
@@ -0,0 +1,69 @@
...
+# Display Conventions and Configuration
+
+No specific display conventions and configuration.
+
+
+# Methods
+
+Sample: NA24143
+
+Lims-id: ACC6356A1
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_lymphoma_6.1.md:19]:
```suggestion
Internal LIMS ID: ACC6356A1
```

File: ucsc_track_hub/hg19/docs/BigWig_lymphoma_6.1.md [hunk 24]
```diff
@@ -0,0 +1,69 @@
...
+
+Case-id: solidcollie
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_lymphoma_6.1.md:21]:
```suggestion
Internal case ID: solidcollie
```

File: ucsc_track_hub/hg19/docs/BigWig_lymphoma_6.1.md [hunk 25]
```diff
@@ -0,0 +1,69 @@
...
+
+Lab Method: KAPA + Twist
+
+Amount reads used for the analysis: 12M r-p
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_lymphoma_6.1.md:25]:
```suggestion
Amount of reads used for the analysis: 12M r-p
```

File: ucsc_track_hub/hg19/docs/BigWig_lymphoma_6.1.md [hunk 26]
```diff
@@ -0,0 +1,69 @@
...
+
+Bioinformatic method: Analyzed with Balsamic 4.0.0 and .bw-file is created with mosdepth
+
+### How the file was generated:
+
+Below procedure can be used for several different panel-designs.
+
+Activate an conda environment with following tools: mosdepth and bedGraphToBigWig. E.g: `source activate D_BALSAMIC-base_develop`
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_lymphoma_6.1.md:33]:
```suggestion
Activate a Conda environment with following tools: mosdepth and bedGraphToBigWig. E.g: `source activate D_BALSAMIC-base_develop`
```

File: ucsc_track_hub/hg19/docs/BigWig_lymphoma_6.1.md [hunk 27]
```diff
@@ -0,0 +1,69 @@
...
+
+2) sort the copied file: `cat <cp.bedgraph> | sort -k1,1 -k2,2n > <cp.sorted.bedgraph>`
+
+3) Run: `bedGraphToBigWig <cp.sorted.bedgraph> /home/proj/production/cancer/reference/GRCh37/genome/hg19.chrom.sizes <output.bw>`.
+
+
+
+
+# External links
+
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_lymphoma_6.1.md:62]:
```suggestion

Read more about BALSAMIC: https://github.com/Clinical-Genomics/BALSAMIC
```

File: ucsc_track_hub/hg19/docs/BigWig_lymphomatic_2.1.md [hunk 28]
```diff
@@ -0,0 +1,69 @@
...
+# Display Conventions and Configuration
+
+No specific display conventions and configuration.
+
+
+# Methods
+
+Sample: Mix from several different HapMap samples
+
+Lims-id: ACC5403A1
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_lymphomatic_2.1.md:19]:
```suggestion
Internal LIMS ID: ACC5403A1
```

File: ucsc_track_hub/hg19/docs/BigWig_lymphomatic_2.1.md [hunk 29]
```diff
@@ -0,0 +1,69 @@
...
+
+Case-id: hipchow_10M
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_lymphomatic_2.1.md:21]:
```suggestion
Internal case ID: hipchow_10M
```

File: ucsc_track_hub/hg19/docs/BigWig_lymphomatic_2.1.md [hunk 30]
```diff
@@ -0,0 +1,69 @@
...
+
+Lab Method: Twist (both lib prep and hybridization)
+
+Amount reads used for the analysis: 10M r-p
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_lymphomatic_2.1.md:25]:
```suggestion
Amount of reads used for the analysis: 10M r-p
```

File: ucsc_track_hub/hg19/docs/BigWig_lymphomatic_2.1.md [hunk 31]
```diff
@@ -0,0 +1,69 @@
...
+
+Bioinformatic method: Analyzed with Balsamic 4.2.4 and .bw-file is created with mosdepth
+
+### How the file was generated:
+
+Below procedure can be used for several different panel-designs.
+
+Activate an conda environment with following tools: mosdepth and bedGraphToBigWig. E.g: `source activate D_BALSAMIC-base_develop`
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_lymphomatic_2.1.md:33]:
```suggestion
Activate a Conda environment with following tools: mosdepth and bedGraphToBigWig. E.g: `source activate D_BALSAMIC-base_develop`
```

File: ucsc_track_hub/hg19/docs/BigWig_lymphomatic_2.1.md [hunk 32]
```diff
@@ -0,0 +1,69 @@
...
+
+2) sort the copied file: `cat <cp.bedgraph> | sort -k1,1 -k2,2n > <cp.sorted.bedgraph>`
+
+3) Run: `bedGraphToBigWig <cp.sorted.bedgraph> /home/proj/production/cancer/reference/GRCh37/genome/hg19.chrom.sizes <output.bw>`.
+
+
+
+
+# External links
+
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_lymphomatic_2.1.md:62]:
```suggestion

Read more about BALSAMIC: https://github.com/Clinical-Genomics/BALSAMIC
```

**Comment by keyvanelhami** [comment]
@hassanfa I have now committed the suggestions.

**Comment by hassanfa** [comment]
Seems travis is slow. Probably too big of files to git clone. I'll merge it.

**Pull request merged**